airflow/utils/db.py

```diff
@@ -238,6 +238,10 @@ def initdb():
                     ]
                 }
             '''))
+    merge_conn(
```

Comment:
Personally I think the default connection list is becoming overbearing and unnecessary, but in addition I think we shouldn't create connections for `contrib` modules. I'd vote to remove this entirely.